Applet/AndroidSEProvider/src/com/android/javacard/keymaster/KMAttestationCertImpl.java

```diff
@@ -106,19 +106,19 @@ public class KMAttestationCertImpl implements KMAttestationCert {
   private static final short[] hwTagIds = {
       KMType.BOOT_PATCH_LEVEL, KMType.VENDOR_PATCH_LEVEL,
       KMType.ATTESTATION_ID_MODEL, KMType.ATTESTATION_ID_MANUFACTURER,
-      KMType.ATTESTATION_ID_MEID, KMType.ATTESTATION_ID_IMEI,
+      KMType.ATTESTATION_ID_MEID, KMType.ATTESTATION_ID_IMEI, KMType.ATTESTATION_ID_SECOND_IMEI,
```

Comment:
These are placed in the order. I think ATTESTATION_ID_SECOND_IMEI should be at the first .
https://cs.android.com/android/platform/superproject/+/master:hardware/interfaces/security/keymint/aidl/android/hardware/security/keymint/KeyCreationResult.aidl;l=212?q=KeyCreationResult.aidl&ss=android%2Fplatform%2Fsuperproject